### PR TITLE
Extend procedures for VMware

### PR DESCRIPTION
**Pull request by maximiliankolb**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.3/Katello 4.5
* [x] Foreman 3.2/Katello 4.4
* [x] Foreman 3.1/Katello 4.3

### CODE REVIEW

**Comment by maximiliankolb** [comment]
Can you please review my PR? @dubewarsagar

File: guides/common/modules/proc_adding-a-vmware-connection-to-server.adoc [hunk 1]
```diff
@@ -4,8 +4,9 @@
 Use this procedure to add a VMware vSphere connection in {ProjectServer}'s compute resources.
 To use the CLI instead of the {ProjectWebUI}, see the xref:cli-adding-vmware-vsphere-connection_{context}[].
 
-Ensure that the host and network-based firewalls are configured to allow {Project} to vCenter communication on TCP port 443.
-Verify that {Project} is able to resolve the host name of vCenter and vCenter is able to resolve {ProjectServer}'s host name.
+.Prerequisites
+* Ensure that the host and network-based firewalls are configured to allow communication from {ProjectServer} to vCenter on TCP port 443.
+* Verify that {ProjectServer} is able to resolve the host name of vCenter and vCenter is able to resolve the host name of {ProjectServer}.
```
Comment (mjivraja) [inline guides/common/modules/proc_adding-a-vmware-connection-to-server.adoc:9]: Ensure that {ProjectServer} and vCenter can resolve each other's host names.

Comment (maximiliankolb) [reply guides/common/modules/proc_adding-a-vmware-connection-to-server.adoc]: Applied!

**Comment by maximiliankolb** [comment]
Ping @dubewarsagar

File: guides/common/modules/proc_adding-vmware-details-to-a-compute-profile.adoc [hunk 2]
```diff
@@ -3,37 +3,45 @@
...
+. In the *Cluster* list, select the name of the target host cluster on the VMware environment.
 . From the *Resource pool* list, select an available resource allocations for the host.
-. In the *Folder* field, enter the folder to organize the host.
+. In the *Folder* list, select the folder to organize the host.
 . From the *Guest OS* list, select the operating system you want to use in VMware vSphere.
-. From the *SCSI controller* list, select the disk access method for the host.
 . From the *Virtual H/W version* list, select the underlying VMware hardware abstraction to use for virtual machines.
-. You can select the *Memory hot add* or *CPU hot add* check boxes if you want to add more resources while the virtual machine is powered.
-. From the *Image* list, select the image to use if performing image-based provisioning.
+. If you want to add more memory while the virtual machine is powered on, select the *Memory hot add* check box.
```
Comment (dubewarsagar) [inline guides/common/modules/proc_adding-vmware-details-to-a-compute-profile.adoc:22]: As per the IBM styleguide, this should be used as a single word. Please see page 305 ibm-style-documentation.

**Comment by dubewarsagar** [comment]
> Can you please review my PR? @dubewarsagar

I have added my suggestion. Rest looks good to me. 
Apologies for delayed response. There was some issue in the notification settings. I have fixed that too. 

Thank you ;)

File: guides/common/modules/proc_adding-vmware-details-to-a-compute-profile.adoc (hunk 2 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_adding-vmware-details-to-a-compute-profile.adoc:22]: Do you mean "check box"? My version of IBM style guide explicitely says "two words":

Screenshot 2022-07-27 at 11 25 33


```
$ git grep "check box" | grep -v "drafts/doc-10-steps/" | wc
     120    2199   21555
$ git grep "checkbox"  | grep -v "drafts/doc-10-steps/" | wc
      12     189    1915
```

Comment (dubewarsagar) [reply guides/common/modules/proc_adding-vmware-details-to-a-compute-profile.adoc:22]: Nope, I mean checkbox, as a single word.

Comment (maximiliankolb) [reply guides/common/modules/proc_adding-vmware-details-to-a-compute-profile.adoc:22]: OK, but IBM style guide says it's _two_ words. Does your version say it's one word?

Comment (dubewarsagar) [reply guides/common/modules/proc_adding-vmware-details-to-a-compute-profile.adoc:22]: Yes!

Comment (maximiliankolb) [reply guides/common/modules/proc_adding-vmware-details-to-a-compute-profile.adoc]: Thanks for letting me know! Sadly IBM style guide is not open source/freely available ... 👀

**Comment by maximiliankolb** [comment]
@dubewarsagar Please re-review; should be good now.

**Comment by dubewarsagar** [comment]
> @dubewarsagar Please re-review; should be good now.

Looks good! Thank you :)

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked: ✅ 
   a8d4e317b..015ba3da4  3.3 -> 3.3
   53a75b764..214a2f6a9  3.2 -> 3.2
   e251c4eb6..633e067d8  3.1 -> 3.1

**Pull request merged**